src/main/java/bowling/domain/frame/Frames.java

```diff
@@ -39,14 +39,18 @@ public int nextFrameNo() {
         return this.frames.size();
     }
 
-    public boolean isFinish() {
+    public boolean finish() {
         return frames.size() == FINAL_FRAME && lastFrame().finish();
     }
 
     public List<Frame> getFrames() {
         return frames;
     }
 
+    public boolean turnFinish(){
+        return lastFrame().finish();
+    }
```

Comment:
```suggestion
    public boolean finish() {
        return frames.size() == FINAL_FRAME && turnFinish();
    }

    public List<Frame> getFrames() {
        return frames;
    }

    public boolean turnFinish(){
        return lastFrame().finish();
    }
```
이렇게 만들어놓은 함수들을 적극적으로 사용해주면 더 좋을 것 같아요!